integrations/instructor_embedders/src/haystack_integrations/components/embedders/instructor_embedders/instructor_document_embedder.py

```diff
@@ -62,7 +63,7 @@ def __init__(
         self,
         model: str = "hkunlp/instructor-base",
         device: Optional[str] = None,
-        use_auth_token: Union[bool, str, None] = None,
+        token: Optional[Secret] = Secret.from_env_var("HF_API_TOKEN", strict=False),  # noqa: B008
```

Comment:
that's actually a better idea, to have those `ignore` statements inline, instead of globally defined 👍🏽